api/pyproject.toml

```diff
@@ -0,0 +1,208 @@
+[build-system]
+requires = ["hatchling", "hatch-openzim"]
+build-backend = "hatchling.build"
+
+[project]
+name = "zimitfrontend"
+requires-python = ">=3.12,<3.13"
+description = "Backend For Frontend API for zimit-frontend UI"
+readme = "README.md"
+dependencies = [
+    "humanfriendly==10.0",
+    "fastapi[all]==0.114.0",
+    "pydantic==2.9.1",       # this is also a sub-dep of fastapi but we rely a lot on it
+    "pyhumps==3.8.0",
+    "requests==2.32.3",
+    "python-i18n==0.3.9",
```

Comment:
I completely forgot to check this, thank you.

I agree this is a concern, but:
- there is not really an alternative from my PoV, gettext / Babel seem to be really complex for our simple need, and not really compatible with JSON
- there are already forks available, maybe one will achieve to merge one day or the other
- the codebase is really very limited, shouldn't be a big concern to maintain on our own should things go wild

Anyway I've quite isolated the translation module so that we can easily switch from one to another if needed.

I propose to continue with this module for now, and decide in 6 months / 1 year if we need to change something. WDYT?